src/applications/vaos/appointment-list/components/NoAppointments.jsx

```diff
@@ -5,6 +5,7 @@ import NewTabAnchor from '../../components/NewTabAnchor';
 export default function NoAppointments({
   showScheduleButton,
   startNewAppointmentFlow,
+  additionalDescription = false,
```

Comment:
instead of setting the default prop here, we can declare it below the  propTypes like:
```
NoAppointments.defaultProps = {
  additionalDescription: false,
};

```

Reply:
fixed 👍

Reply:
Hey, I think you misunderstood my earlier comment. This proptype should be PropTypes.bool.
I suggested adding an additional line of code below `NoAppointments. propTypes`. After defining the propTypes, we can define defaultProp values like this:
```
NoAppointments.defaultProps = {
  additionalDescription: false,
};

```

Reply:
I think I have fixed this but I am not positive

Reply:
That looks good! last suggestion I have is; `additionalRequestDescription` is probably good but `showAdditionalRequestDescription` is a little more descriptive of the purpose of the prop.

Reply:
Ok I have just fixed this issue